### DISCUSSION TITLE
Preset documentation contradicts each other

### DISCUSSION THREAD

**Discussion by Sebazzz** [renovatebot/renovate#28434] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitea

### Please tell us more about your question or problem

In the [Sharable Config Presets](https://docs.renovatebot.com/config-presets/) documentation is stated:

> **Warning**
> Only use default.json for your presets.

As I understand, this means that we should only use the filename "default.json", so not "default.json5" and not "my-preset.json" in our custom preset repositories. 

However, at the [examples](https://docs.renovatebot.com/config-presets/#self-hosted-git) various examples are stated where the file-name is *not* `default.json`:

![image](https://github.com/renovatebot/renovate/assets/1426097/9266665c-8e4c-44da-8440-cb4ba04e57b6)

What is true?

**Comment by ceng-p** [comment]
I interpret it as you should not use "renovate.json" anymore for your presets. But I agree it's a bit misleading as you can name them as you want, aslong as you specify the right filename in your usage.

**Comment by rarkins** [comment]
Clarified here: https://github.com/renovatebot/renovate/pull/28443

**Reply by Sebazzz** [reply]
Great! 😀

So if you want to use json5 in a preset, you explicitly need to reference it with the full file name.

**Reply by viceice** [reply]
yes
